README.md

```diff
@@ -669,6 +670,32 @@ Now we can proceed as with Spike, debugging works in a similar way:
 
 Further information about GDB debugging is available [here](https://sourceware.org/gdb/onlinedocs/gdb/) and [here](https://sourceware.org/gdb/onlinedocs/gdb/Remote-Debugging.html#Remote-Debugging).
 
+## <a name="ide"></a> Building Rocket Chip with an IDE
+
+The Rocket Chip Scala build uses the standard Scala build tool SBT.
+IDEs like [IntelliJ](https://www.jetbrains.com/idea/) and [VSCode](https://code.visualstudio.com/)
+are popular in the Scala community and work with Rocket Chip.
+To use one of these IDEs, there is one minor peculiarity of the Rocket Chip build that must be addressed.
+
+In order to support both building against both source and published version of Chisel3 and FIRRTL,
+Rocket Chip uses an SBT plugin that it configures via JVM system properties.
+This flow is described in the [Chisel README](https://github.com/freechipsproject/chisel3#building-chisel-with-firrtl-in-the-same-sbt-project).
+
+When a file named `.sbtopts` exists in the root of this repository, it means Rocket Chip is configured to build Chisel3 and FIRRTL from source.
+This file works when invoking SBT in the shell, but because it uses an environment variable `$PWD`, it is not interpreted correctly by IDEs.
+One can fix this issue by using IDE-specific import settings to override the `sbt.workspace` JVM system property to point to the root directory of Rocket Chip with an absolute path, or one can edit the file to expand the variable.
+
+This can be done in `bash` simply as follows:
+
+```bash
+sed -i "s|\$PWD|$PWD|" .sbtopts
+```
+
+So long as this is set properly, the Rocket Chip SBT project should work in any IDE that supports SBT.
```

Comment:
So long as this is set properly, or `.sbtopts` does not exist, ... ?

Reply:
Yeah that's right, I'll clarify the text